src/peergos/shared/NetworkAccess.java

```diff
@@ -299,6 +327,25 @@ public static NetworkAccess build(ContentAddressedStorage dht,
         });
     }
 
+    public CompletableFuture<Optional<FileWrapper>> getFile(WriterData base,
+                                                            AbsoluteCapability cap,
+                                                            Optional<SigningPrivateKeyAndPublicHash> entryWriter,
+                                                            String ownerName) {
+        return getMetadata(base, cap)
+                .thenApply(faOpt -> faOpt.map(fa -> new FileWrapper(Optional.empty(),
```

Comment:
Since we no longer have FileAccess, DirAccess classes var references to faOpt, fa don't mean as much. But i have noticed fileAccess is used in many places

Reply:
Semantically they are still as they were - file and directory accesses. They are functionally different. It's just that they have the same serialization now. I certainly still think of them as dir access and file access.